gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorage.java

```diff
@@ -419,7 +419,10 @@ void compose(String bucketName, List<String> sources, String destination, String
    *
    * @param key name of the object related statistics key
    */
-  public AtomicLong getObjectStatistics(GoogleCloudStorageStatistics key);
+  public AtomicLong getStatistics(GoogleCloudStorageStatistics key);
+
+  /** To get the http requests tracker */
+  public GcsioTrackingHttpRequestInitializer getGcsRequestsTracker();
```

Comment:
Instead of returing request initializer, can we return stats instead

Reply:
Updated